corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -315,13 +316,14 @@ <h3 class="panel-title">{% trans 'Mobile Workers' %}</h3>
           {% endif %}
         </div>
         <table class="table table-striped table-responsive" style="margin-bottom: 0;" data-bind="visible: showTable">
+          {% with two_stage_user_confirmation=request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING_BY_SMS" %}
```

Comment:
@orangejenny `two_stage_user_confirmation` should be OR of flags TWO_STAGE_USER_PROVISIONING_BY_SMS and TWO_STAGE_USER_PROVISIONING but it gives syntax error for these:

`{% with two_stage_user_confirmation=request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING_BY_SMS" 
 or request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING" %}`
and
`{% with two_stage_user_confirmation=request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING_BY_SMS" 
 || request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING" %}`

Any suggestion what could be used here?

Reply:
I think that logic is too complex for the template and you need to create a context variable in the view.

Reply:
ok